### DISCUSSION TITLE
Using the Curiosity Nano 1607

### DISCUSSION THREAD

**Discussion by stevecolt** [SpenceKonde/DxCore#121] [Q&A]
Having not had much luck with serialUDPI I was very excited to discover the curiosity nano board with a 1607 built in updi programmer and serial port.
So I got a board, installed drivers, and tried a blink program. Fails with bad response to avr sign on as shown in attached listing.
Do I need to do anything else with the curiosity, I don't know load it MPLAB-X to update or register on my machine, or is it plug and play. My apologies if there is a guide, I followed whatever I found, but may have missed something
[1607CuriosityError.txt](https://github.com/SpenceKonde/DxCore/files/6676675/1607CuriosityError.txt)
Thanks
Steve

**Comment by stevecolt** [comment]
Think Ive managed to get myself totally confused. DXCore isn't for 1607 is it?
Soon as I entered RIGHtTpart in MegaTinyCore the Curiosity works fine as a programmer and the serial port seems to work as default serial.
A Minor issue is that you (maybe that should read I)  cant set the target volts by defining a CMD in a text file on the curiosity drive, it just ignores it, and then deletes from drive. not a big deal as easily strapped.
So that seems to be a very reliable way of working and debugging, 
Now to get on with some coding!!!

Sorry if anyone wasted time looking at post, but maybe someone else might like to know it does work without any fiddling.
Steve

**Comment by SpenceKonde** [comment]
Glad it's sorted. I don't know nothin about how to that voltage sdetting is handled.
